frame/staking/src/lib.rs

```diff
@@ -3127,6 +3139,42 @@ impl<T: Trait> Module<T> {
 		})
 	}
 
+	/// Use a damping function to update the target number of validators for the next selection.
+	fn update_target_validators(mut exposures: Vec<BalanceOf<T>>) -> u32 {
```

Comment:
the name exposure should ideally refer to the type `Exposure`, maybe call this `exposed_balances`